lib/attr_json/type/container_attribute.rb

```diff
@@ -1,14 +1,16 @@
 module AttrJson
   module Type
-    # A type that gets applied to the AR container/store jsonb attribute,
-    # to do serialization/deserialization/cast using declared attr_jsons, to
-    # json-able values, before calling super to original json-type, which will
-    # actually serialize/deserialize the json.
-    class ContainerAttribute < (if Gem.loaded_specs["activerecord"].version.release >= Gem::Version.new('5.2')
+    SuperKlass = (if Gem.loaded_specs["activerecord"].version.release >= Gem::Version.new('5.2')
       ActiveRecord::Type::Json
     else
       ActiveRecord::Type::Internal::AbstractJson
     end)
+
+    # A type that gets applied to the AR container/store jsonb attribute,
+    # to do serialization/deserialization/cast using declared attr_jsons, to
+    # json-able values, before calling super to original json-type, which will
+    # actually serialize/deserialize the json.
+    class ContainerAttribute < AttrJson::Type::SuperKlass
```

Comment:
I don't understand what this is doing. `SuperKlass` is either `ActiveRecord::Type::Json` or `ActiveRecord::Type::Internal::AbstractJson` (above). 

So what is `AttrJson::Type::SuperKlass`?  Does that turn into `AttrJson::Type::ActiveRecord::Internal::AbstractJson` or `AttrJson::Type::ActiveRecord::Type::Json`? I don't think so -- becuase those don't exist!

I'm wondering if it actually ends up being nothing but the same as `SuperKlass`, and no actualy change to previous code before this diff?

What were you actually trying to do here?

Reply:
Oh this is from when I was trying to make the custom type work. I didn’t actually change the super class logic. So we can revert this part.